src/Maestro/Microsoft.DotNet.Maestro.Tasks/src/PushMetadataToBuildAssetRegistry.cs

```diff
@@ -393,7 +404,7 @@ private string GetVersion(string assetId)
                 }
             }
 
-            return (buildsManifestMetadata, signingInfo, manifestBuildData);
+            return (buildsManifestMetadata, signingInfo, manifestBuildData, parsedManifests);
```

Comment:
It doesn't look like you're doing anything to the ParsedManifest list in this method, does it really need to return it, when the caller will already always have it ready anyways?

Reply:
this was outdated code